presets/shared.yml

```diff
@@ -99,7 +99,8 @@ finalizationSize: 10000
 finalizationThreshold: 7000
 maxHashesPerPoint: 256
 prevoteBlocksMultiple: 4
-unfinalizedBlocksDuration: 0m
+nonVotingUnfinalizedBlocksDuration: 0m
+votingUnfinalizedBlocksDuration: 8m
```

Comment:
When the node is voting - votingUnfinalizedBlocksDuration: 0m
Nonvoting node - nonVotingUnfinalizedBlocksDuration: 8m